app/routing/ArrivalsRouter.scala

```diff
@@ -50,17 +50,15 @@ class ArrivalsRouter @Inject() (
     with StreamingParsers
     with VersionedRouting {
 
-  val VERSION_2_ACCEPT_HEADER_PATTERN = """^application\/vnd\.hmrc\.2\.0\+.+$""".r
-
   def createArrivalNotification(): Action[Source[ByteString, _]] =
     route {
-      case Some(VersionedRouting.VERSION_2_ACCEPT_HEADER_VALUE_JSON) => v2Arrivals.createMovement(MovementType.Arrival)
-      case _                                                         => v1Arrivals.createArrivalNotification()
+      case Some(VersionedRouting.VERSION_2_ACCEPT_HEADER_PATTERN()) => v2Arrivals.createMovement(MovementType.Arrival)
```

Comment:
For all endpoints besides the Get single message endpoint, we return the JSON so the only accept header we should allow is this one VERSION_2_ACCEPT_HEADER_VALUE_JSON. 

For the Get single message endpoint, the user can specify whether they want their response in json with the message in XML, or just in JSON where we convert their XML message to JSON.

This isn't the case for all other endpoints since all the data returned is solely json. This change here is correct, since we want to route to version 2 if they have used the version 2 pattern for the accept header, but we need more refinement in the action to determine whether the endpoint accepts JSON or also JSON wrapped XML